synchronize.py

```diff
@@ -0,0 +1,156 @@
+from __future__ import print_function
+from datetime import datetime
+import pickle
+import os.path
+from googleapiclient.discovery import build
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+from aw_core.models import Event
+from aw_client import ActivityWatchClient
+
+
+# If modifying these scopes, delete the file token.pickle.
+SCOPES = ['https://www.googleapis.com/auth/calendar.readonly']
+calendarId = 'primary'
+
+def get_token():
+    try:
+        with open('token.txt','r') as f:
+            token = f.readline()
+    except FileNotFoundError:
+        token = None
+    return token
+
+def store_token(token):
+    with open('token.txt','w') as f:
+        f.write(token)
```

Comment:
For now the synchronization token is stored and retrieved from a regular text file. I'm not sure if it justifies creating more complexity, such as adding a completely new table in the database for it. It also doesn't need to be stored securely, since all the information it contains is at which point in time you last synchronized, and doesn't contain any actual information about the events themselves.